<launch>
    <node pkg="explorefzh" type="explorefzh_node" name="explorefzh_node" respawn="false"
        output="screen">
        <param name="robot_base_frame" value="base_footprint" />
        <param name="costmap_topic" value="map" />
        <param name="costmap_updates_topic" value="map_updates" />
        <param name="visualize" value="true" />
        <param name="planner_frequency" value="0.33" />
        <param name="progress_timeout" value="30.0" />

        <!-- 距离权重 -->
        <!-- <param name="potential_scale" value="3.0" /> -->
        <!-- 方向权重 -->
        <!-- <param name="orientation_scale" value="0.0" /> -->
        <!-- 边界大小权重 -->
        <!-- <param name="gain_scale" value="1.0" /> -->

        <param name="potential_scale" value="0.4" />
        <param name="orientation_scale" value="0.2" />
        <param name="gain_scale" value="0.4" />

        <param name="transform_tolerance" value="0.3" />
        <param name="min_frontier_size" value="0.75" />

        <!-- <param name="dbscan_epsilon" value="10.0"/> -->
        <!-- <param name="dbscan_minPts" value="2.0"/> -->

    </node>
</launch>